pkg/config/config.go

```diff
@@ -290,7 +290,7 @@ func init() {
 	BindEnvAndSetDefault("hpa_watcher_gc_period", 60*5) // 5 minutes
 	BindEnvAndSetDefault("external_metrics_provider.enabled", false)
 	BindEnvAndSetDefault("hpa_configmap_name", "datadog-custom-metrics")
-	BindEnvAndSetDefault("external_metrics_provider.batch_window", 5) // 5 seconds to batch calls to the GlobalStore
+	BindEnvAndSetDefault("external_metrics_provider.batch_window_configmap", 5) // 5 seconds to batch calls to the GlobalStore
```

Comment:
I only meant to re-word them comment and leave the `external_metrics_provider.batch_window` unchanged. I realize now that I made that very unclear :P .